homeassistant/components/zha/core/registries.py

```diff
@@ -115,7 +115,17 @@ def get_deconz_radio():
         zha.DeviceType.LEVEL_CONTROLLABLE_OUTPUT: LIGHT,
         zha.DeviceType.ON_OFF_LIGHT: LIGHT,
         zha.DeviceType.DIMMABLE_LIGHT: LIGHT,
-        zha.DeviceType.COLOR_DIMMABLE_LIGHT: LIGHT
+        zha.DeviceType.COLOR_DIMMABLE_LIGHT: LIGHT,
+        zha.DeviceType.ON_OFF_LIGHT_SWITCH: BINARY_SENSOR,
+        zha.DeviceType.DIMMER_SWITCH: BINARY_SENSOR,
+        zha.DeviceType.COLOR_DIMMER_SWITCH: BINARY_SENSOR,
+        zha.DeviceType.ON_OFF_BALLAST: SWITCH,
+        zha.DeviceType.DIMMABLE_BALLAST: LIGHT,
+        zha.DeviceType.ON_OFF_PLUG_IN_UNIT: SWITCH,
+        zha.DeviceType.DIMMABLE_PLUG_IN_UNIT: LIGHT,
+        zha.DeviceType.COLOR_TEMPERATURE_LIGHT: LIGHT,
+        zha.DeviceType.EXTENDED_COLOR_LIGHT: LIGHT,
+        zha.DeviceType.ON_OFF_SENSOR: BINARY_SENSOR
```

Comment:
Remove this

Reply:
What should sensors map to? Or how does this work for sensors?

Reply:
Remote devices don’t need the mappings. They emit events.

Reply:
Ah sorry, makes total sense. Will fix this!

Reply:
no need to apologize. We just made this change :)